modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -135,3 +156,15 @@ endif::restricted[]
 ifeval::["{context}" == "installing-restricted-networks-bare-metal"]
 :!restricted:
 endif::[]
+ifdef::openshift-origin[]
```

Comment:
It looks like this ifdef got added probably during a conflict from 4.4, but it wasn't defined at the top of the file.

Reply:
You are right, fixed.